src/main/index.js

```diff
@@ -68,7 +86,26 @@ app.on('activate', () => {
   if (!rendererWindow) rendererWindow = createWindow();
 });
 
-app.on('will-quit', () => {
+app.on('will-quit', (e) => {
```

Comment:
nit: use a more descriptive name (http://airbnb.io/javascript/#naming--descriptive)

Reply:
Does this apply even to `e`? That has a standard meaning of "event," similar to `x`, `y`, `i`, etc.

If we want to be even more specific than "event," it would have to be `willQuitEvent` or something, which sounds redundant

Reply:
I think `event` would fit best here. We don't have multiple events in this callback function so no need to specify precisely the event, and it would also be redundant as you pointed it out. Anyway, this is a nit.